tests/src/components/Card.test.js

```diff
@@ -20,3 +20,10 @@ test('CardList snapshot test', () => {
   expect(component).toMatchSnapshot();
   expect(component2).toMatchSnapshot();
 });
+
+test('CardList snapshot test', () => {
+  const component = shallow(<CardList article={articles} className="category-cont" title="testing" />);
+  const component2 = shallow(<CardList article={articles} className="testing" title="testing" />);
+  expect(component).toMatchSnapshot();
+  expect(component2).toMatchSnapshot();
```

Comment:
Is there any reason why you named this variable: `component2`. Please use an appropriate variable name. Refer also to this: [https://github.com/airbnb/javascript/tree/es5-deprecated/es5#naming-conventions](url)

Reply:
I have made these adjustments.